updating migration guide (#286)
infinum/eightshift-docs: website/docs/migrations/5-6.md

- [eightshift/frontend-libs](https://github.com/infinum/eightshift-frontend-libs/releases/tag/5.0.0) - **4+ --> 5.0.0**
- [eightshift/libs](https://github.com/infinum/eightshift-libs/releases/tag/4.0.0) - **3+ --> 4.0.0**

> ## JavaScrip changes:

Reviewer: ```suggestion
## JavaScript changes
```

> Block > Component - need to pass setAttributes

Reviewer: ```suggestion
Block → Component - need to pass `setAttributes`
```

> Component > Component - no need to pass setAttributes or anything other that you are not overriding.

Reviewer: ```suggestion
Component → Component - no need to pass `setAttributes` or anything that you're not overriding.
```

- Old usage: `Blocks::props($attributes, 'button', 'new')`
- New usage: `Components::props('new', $attributes)`

## Help

> We made this few regex's to help you with the migrations.

Reviewer: ```suggestion
We prepared a couple of regular expressions (_regexes_) to help with the migrations.
```
